x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/assets.tsx

```diff
@@ -231,6 +241,13 @@ export const AssetsPage = ({ packageInfo, refetchPackageInfo }: AssetsPanelProps
                 }}
```

Comment:
is there already a docs issue to update https://www.elastic.co/guide/en/fleet/current/install-uninstall-integration-assets.html once this feature is enabled by default? if not, let's go ahead and create a placeholder issue

Reply:
No there is not, I will create one for the whole space awareness project as there is probably more than this to document.